composer.json

```diff
@@ -27,12 +27,12 @@
     }
   ],
   "require": {
-    "php": ">=5.5",
-    "guzzlehttp/guzzle": "~6.0@dev|>=6",
-    "psr/log": "~1|~1.0|1.0.0|dev-master"
+    "php": "^5.5.9 || ^7.0",
```

Comment:
5.5.9 ships with ubuntu LTS, and supporting less than that is just as bad as still supporting php 5.3 tbh